Auto Layout/CollectionViewController.swift

```diff
@@ -57,25 +87,47 @@ final private class CollectionViewDataSource: NSObject, UICollectionViewDataSour
     self.cellCount = cellCount
   }
 
-  func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+  func numberOfSections(in collectionView: UICollectionView) -> Int {
     return cellCount
   }
 
+  func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+    if section.isCarouselSection {
+      return 10
+    }
+    return 1
```

Comment:
we could ternary this just for a little bit less code, unless you were thinking we might need other cases here in the future? if so, then ignore this comment
```suggestion
    return section.isCarouselSection ? 10 :  1
```